cli/config/configuration-fc.toml.in

```diff
@@ -289,9 +289,11 @@ path = "@NETMONPATH@"
 # the container network interface
```

Comment:
could you please also update the acrn file (configuration-acrn.toml.in) ?

Reply:
done